central/notifiers/cscc/cscc.go

```diff
@@ -232,7 +234,72 @@ func (c *cscc) AlertNotify(ctx context.Context, alert *storage.Alert) error {
 	)
 }
 
-func newCSCC(protoNotifier *storage.Notifier) (*cscc, error) {
+// AlertNotify takes in an alert and generates the notification
```

Comment:
Change comment to reflect `InitFinding` purpose

Reply:
done